src/controllers/mainController.ts

```diff
@@ -870,16 +880,31 @@ export default class MainController implements vscode.Disposable {
 	/**
 	 * Prompt the user to view release notes if this is new extension install
 	 */
-	private async showReleaseNotesPrompt(): Promise<void> {
+	private async showFirstLaunchPrompts(): Promise<void> {
 		let self = this;
 		if (!this.doesExtensionLaunchedFileExist()) {
-			// ask the user to view a scenario document
-			let confirmText = 'View Now';
-			const choice = await this._vscodeWrapper.showInformationMessage(
-				'View mssql for Visual Studio Code release notes?', confirmText);
-			if (choice === confirmText) {
-				await self.launchReleaseNotesPage();
-			}
+			// ask the user to view release notes document
+			let confirmText = LocalizedConstants.viewMore;
+			let promiseReleaseNotes = this._vscodeWrapper.showInformationMessage(
+				LocalizedConstants.releaseNotesPromptDescription, confirmText)
+				.then(async (result) => {
+					if (result === confirmText) {
+						await self.launchReleaseNotesPage();
+					}
+				});
+
+
+			// ask the user to view encryption changes document
+			let confirmTextEncrypt = LocalizedConstants.moreInformation;
```

Comment:
Don't we want to show this even for users who already had the extension installed?